src/views/EdgeNode/FormFields/FormFieldsDrawerService.vue

```diff
@@ -45,26 +45,20 @@
       <div class="flex flex-col w-full sm:max-w-3xl gap-2">
         <div class="flex flex-col gap-2">
           <div class="flex w-80 sm:max-w-lg flex-col items-start gap-2">
-            <label
-              for="name"
-              class="text-color text-sm not-italic font-medium leading-5"
-              >Service *</label
-            >
-            <Dropdown
-              filter
-              autoFilterFocus
-              appendTo="self"
-              class="flex self-stretch"
-              v-model="service"
+            <FieldDropdown
+              label="Service *"
+              name="service"
               :options="props.listServices"
               :loading="props.listServices.length === 0"
+              @change="resetRegionAndCity"
```

Comment:
Essa alteração está correta?
```suggestion
              @change="resetRegionAndCity"
```